arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java

```diff
@@ -388,31 +389,37 @@ private void populateColumns(final MessageFrame frame) {
                 EWord.of(frame.getStackItem(0)), BigInteger.valueOf(frame.getInputData().size()));
         oobParameters = cdlOobParameters;
         setCdl(cdlOobParameters);
+      } else if (isSstore) {
+        final SstoreOobParameters sstoreOobParameters =
+            new SstoreOobParameters(BigInteger.valueOf(frame.getRemainingGas()));
+
+        oobParameters = sstoreOobParameters;
+        setSstore(sstoreOobParameters);
+      } else if (isDeployment) {
+        final DeploymentOobParameters deploymentOobParameters =
+            new DeploymentOobParameters(EWord.of(frame.getStackItem(0)));
+
+        oobParameters = deploymentOobParameters;
+        setDeployment(deploymentOobParameters);
       } else if (isXCall) {
-        // CallOobParameters is used since this is a subcase of CALL
-        CallOobParameters callOobParameters =
-            new CallOobParameters(
-                EWord.of(frame.getStackItem(2)),
-                BigInteger.ZERO,
-                !frame.getStackItem(2).isZero(),
-                BigInteger.ZERO);
-        oobParameters = callOobParameters;
-        setXCall(callOobParameters);
+        XCallOobParameters xCallOobParameters =
+            new XCallOobParameters(EWord.of(frame.getStackItem(2)));
+        oobParameters = xCallOobParameters;
+        setXCall(xCallOobParameters);
       } else if (isCall) {
         final Account callerAccount = frame.getWorldUpdater().get(frame.getRecipientAddress());
         // DELEGATECALL, STATICCALL cases
-        EWord val = EWord.ZERO;
+        EWord value = EWord.ZERO;
         boolean nonZeroValue = false;
         // CALL, CALLCODE cases
         if (opCode == OpCode.CALL || opCode == OpCode.CALLCODE) {
-          val = EWord.of(frame.getStackItem(2));
+          value = EWord.of(frame.getStackItem(2));
```

Comment:
could be `final Eword value = (opCode == OpCode.CALL || opCode == OpCode.CALLCODE) ? EWord.of(frame.getStackItem(2)) :  EWord.ZERO;

Reply:
I believe it is a bit more readable as it is